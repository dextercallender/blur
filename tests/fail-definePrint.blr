void print() {
	int p;	
}

void main() {
	int x = 7;	
}