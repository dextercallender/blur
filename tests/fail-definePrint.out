Fatal error: exception Failure("Cannot overwrite print function!!")
